.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on: [push]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Prepare repository
+        run: git fetch --unshallow --tags
+
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+
+      - name: Cache node modules
+        uses: actions/cache@v1
+        with:
+          path: node_modules
+          key: yarn-deps-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            yarn-deps-${{ hashFiles('yarn.lock') }}
+
+      - name: Create Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
```

Comment:
Are these personal tokens from your account, or do they map back to a particular github/npm account? It may be handy to add a comment pointing to those accounts :)

Reply:
I generated both of these tokens. Mostly to get started, but later realized that NPM tokens can only be generated from an individual account (and maybe Github ones as well? they may have organizational tokens now). Since `secrets` are added in the Github settings anyone with repo access can edit them. I can add comments for the moment to document. I will also touch base with dev-inf to see how they want to manage this.